SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/RPCPayload.h"
+#include "Schema/UnrealObjectRef.h"
+#include "SpatialConstants.h"
+
+#include "UObject/WeakObjectPtr.h"
+#include "CoreMinimal.h"
+
+struct FUnrealObjectRef;
+struct FPendingRPCParams;
```

Comment:
Similarly with this. I don't know what TArray requires though. But generally including something a template parameter only needs a forward declaration. But you need the type to be complete by the time certain functions are called.

Reply:
I believe CoreMinimal should include everything necessary.

Reply:
Core minimal won't include these as they are our types.

Reply:
Could you check the latest revision. I believe that everything that has to be included - is included.